docker/kratos/kratos.yml

```diff
@@ -1,9 +1,9 @@
-version: v1.0.0
+version: v1.1.0
 
 oauth2_provider:
     url: "http://hydra:4445"
 log:
-    level: debug
+    level: trace
```

Comment:
question/possible issue: do we actually need `trace` level logging? This could be truly overwhelming in a production setting. Can't we just do `info`?

Reply:
In next commits this gets changed back to `debug`, still my question stands

Reply:
Since this is a testing setup not intended to be used in production, I think `debug` is acceptable. In kratos charm the log level is configurable: https://github.com/canonical/kratos-operator/blob/7c6973a8fd9172f1fb2f22031f22f2883e234321/config.yaml#L48

Reply:
Oh ok, my bad! I didn't realize